web3swift/src/Extensions/Data+Random.swift

```diff
@@ -10,14 +10,6 @@ import Foundation
 
 extension Data {
     static func randomOfLength(_ length: Int) -> Data? {
-        var data = [UInt8](repeating: 0, count: length)
-        let result = SecRandomCopyBytes(kSecRandomDefault,
-                               data.count,
-                               &data)
-        if result == errSecSuccess {
-            return Data(data)
-        }
-        
-        return nil
+        return Data((0 ..< length).map { _ in UInt8.random(in: UInt8.min ... UInt8.max) })
```

Comment:
I see that the system generator uses `arc4random_buf` in Apple platforms; not 100% sure it has the same security guarantees as `SecRandomCopyBytes`, so to avoid introducing a change here, can we wrap it with `canImport(Security)` to keep the same in these platforms?

As a side note, I think `getRandom(2)` is not good enough for Linux either, but can leave it using the higher level API instead.

Reply:
from `man arc4random`

> The original version of this random number generator used the RC4 (also known as ARC4) algorithm.  In OS X 10.12 it was replaced with the NIST-approved AES cipher, and it may be replaced again in the future as cryptographic techniques advance.  A good mnemonic is “A Replacement Call for Random”.

This change seems to have been introduced on iOS 10 as well, since this framework itself only supports iOS 11 onwards it should be secure, what do you think?

Reply:
ok agreed, thanks for checking